src/main/java/nextstep/subway/application/LineService.java

```diff
@@ -0,0 +1,60 @@
+package nextstep.subway.application;
+
+import nextstep.subway.Exception.NotFoundLineException;
+import nextstep.subway.Exception.NotFoundStationException;
+import nextstep.subway.domain.*;
+import nextstep.subway.dto.LineRequest;
+import nextstep.subway.dto.LineResponse;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional
+public class LineService {
+    private LineRepository lineRepository;
```

Comment:
```suggestion
    private final LineRepository lineRepository;
```
StationService 와 다르게 선언하신 이유가 있으신가요?

Reply:
다 final로 만들려고 했는데 제가 빠뜨린것 같습니다
수정하겠습니다!